test/functional/feature_notifications.py

```diff
@@ -65,6 +66,35 @@ def run_test(self):
         for tx_file in os.listdir(self.walletnotify_dir):
             os.remove(os.path.join(self.walletnotify_dir, tx_file))
 
+        if self.is_wallet_compiled():
+            # Setup the descriptors to be imported to the wallet
+            seed = "cTdGmKFWpbvpKQ7ejrdzqYT2hhjyb3GPHnLAK7wdi5Em67YLwSm9"
```

Comment:
cafc78c0869c1b6712829b8944dfd94b753d07ba : linter complains `test/functional/feature_notifications.py:71:13: F841 local variable 'seed' is assigned to but never used`